src/app/clusters/network-commissioning/network-commissioning-ember.cpp

```diff
@@ -0,0 +1,163 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "network-commissioning.h"
+
+#include <cstring>
+
+#include "af.h"
+#include <gen/callback.h>
+
+#include "gen/att-storage.h"
+#include "gen/attribute-id.h"
+#include "gen/attribute-type.h"
+#include "gen/cluster-id.h"
+#include "gen/command-id.h"
+
+/**
+ * @brief Network Commissioning Cluster AddThreadNetwork Command callback
+ * @param operationalDataset
```

Comment:
These doccomments seem useless to me.

Could we remove them from the template or do they have a chance to contain something useful?

Reply:
They are just implementing callbacks required by the ember library. I think removing this is totally ok, however, I also want to make the network commissioning a template of how to write cluster handlers, so I left some comments here and removed the notes for params -- they are just params for cluster commands.